content/docs/hooks-effect.md

```diff
@@ -333,13 +334,13 @@ function FriendStatusWithCounter(props) {
 }
 ```
 
-**Hooks lets us split the code based on what it is doing** rather than a lifecycle method name. React will apply *every* effect used by the component, in the order they were specified.
+**Hoki pozwalają na dzielenie kodu ze względu na to, za co on odpowiada**, a nie ze względu na nazwę metody cyklu życia. React wywoła *każdy* efekt użyty w komponencie, w kolejności, w jakiej został dodany.
```

Comment:
```suggestion
**Hooki pozwalają na dzielenie kodu na fragmenty względem tego, za co one odpowiadają**, a nie ze względu na nazwę metody cyklu życia. React wywoła *każdy* efekt użyty w komponencie w takiej kolejności, w jakiej został dodany.
```

Reply:
```suggestion
**Hooki pozwalają na dzielenie kodu na mniejsze fragmenty względem ich odpowiedzialności**, a nie ze względu na nazwę metody cyklu życia. React wywoła *każdy* efekt użyty w komponencie w takiej kolejności, w jakiej został dodany.
```

Reply:
To może "pod względem"? Wtedy już w ogóle będzie kul ;-)